python/mxnet/rnn/rnn_cell.py

```diff
@@ -495,10 +505,10 @@ def unpack_weights(self, args):
             weight = args.pop('%s%s_weight'%(self._prefix, group_name))
             bias = args.pop('%s%s_bias' % (self._prefix, group_name))
 
-            for j, name in enumerate(['_z', '_r', '_o']):
-                wname = '%s%s%s_weight' % (self._prefix, group_name, name)
+            for j, gate in enumerate(self._gate_names):
+                wname = '%s%s_%s_weight' % (self._prefix, group_name, gate)
                 args[wname] = weight[j*h:(j+1)*h].copy()
-                bname = '%s%s%s_bias' % (self._prefix, group_name, name)
+                bname = '%s%s_%s_bias' % (self._prefix, group_name, gate)
```

Comment:
Yes. The only part that needs empty gate name is in FusedRNNCell, which took care of it. I think changing the gate names to be without underscore prefix improves readability.